src/pyrocore/data/config/rtorrent-0.8.6.rc

```diff
@@ -39,6 +39,11 @@ system.method.insert = pyro.activations.append,simple|private,"d.set_custom=acti
 system.method.set_key = event.download.paused,activations,"pyro.activations.append=P"
 system.method.set_key = event.download.resumed,activations,"pyro.activations.append=R"
 
+# EVENTS: Timestamp 'tm_downloaded' (time when meta (torrent) file was downloaded)
+method.insert = d.tm_downloaded, simple, "execute.capture=sh,-c,\"echo -n \$(stat -c %Y \\\"$1\\\")\",getDownloadedDate,(d.tied_to_file)"
+method.insert = d.tm_downloaded.set, simple|private, "d.custom.set=tm_downloaded,(cat,(d.tm_downloaded)); d.save_full_session="
```

Comment:
These two should be both private, and named _tm_downloaded_get/init. Then define `d.tm_downloaded` to query the custom variable. Call the `init` helper method in the event handler. Right now, you always ask from disk, which is way more fragile (think untie).